### DISCUSSION TITLE
Discretization of Splines

### DISCUSSION THREAD

**Discussion by EngDM** [robbievanleeuwen/section-properties#283] [Ideas]
Hello,

I've come across some interesting joist chord shapes, and it appears that the center portion is a tear drop shape. Using the import from DXF, it works however it appears that the spline was split into polylines (which is fine) however, is it possible to tell the import to use X number of nodes when doing this, so we can get a more accurate curved shape?

I don't see any mention of splines in the documentation, however I suspect this may be an issue for shapely. I'm not sure if it is currently possible to send any input to shapely to control nodes.

**Comment by robbievanleeuwen** [comment]
Hi @EngDM, thanks for raising this. I believe @ccaprani has implemented this option in his PR #274 which I am hoping to merge shortly!
